bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -104,6 +112,7 @@ public void getTibberParameters() {
         try {
             httpHeader.put("cache-control", "no-cache");
             httpHeader.put("content-type", JSON_CONTENT_TYPE);
+            httpHeader.put("user-agent", "openHAB/Tibber " + versionString + " com.tibber/1.8.3");
             httpHeader.put("Authorization", "Bearer " + tibberConfig.getToken());
```

Comment:
Sorry, missed this place:
```suggestion
            httpHeader.put(HttpHeader.USER_AGENT.asString(), "openHAB/Tibber " + versionString + " com.tibber/1.8.3");
            httpHeader.put(HttpHeader.AUTHORIZATION.asString(), "Bearer " + tibberConfig.getToken());
```